features/step_definitions/civil_journey_steps.rb

```diff
@@ -880,7 +880,7 @@
 end
 
 Then("the delete modal should open") do
-  expect(page).to have_css(".modal-dialog", visible: true)
+  expect(page).to have_css(".modal-dialog", visible: :visible)
```

Comment:
I always thought `visible` was the default, do we need to specify it here?

Reply:
🤷‍♂️ 
I just followed the guides on https://www.rubydoc.info/gems/rubocop-capybara/RuboCop/Cop/Capybara/VisibilityMatcher
```
# good
expect(page).to have_selector('.foo', visible: :visible)
expect(page).to have_css('.foo', visible: :all)
expect(page).to have_link('my link', visible: :hidden)
```

Reply:
the have_css page is a bit wooly
```
#have_css(css, **options, &optional_filter_block) ⇒ Object
RSpec matcher for whether elements(s) matching a given css selector exist
```
css _would_ exist on the page and it's not explicitly saying visible css

Reply:
The docs are like a magical mystery tour, but...

`have_css` calls [`has_css?`, which accepts the all options that `Finders#all` accepts](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara%2FNode%2FMatchers:has_css%3F).

[The `visible` option defaults to `ignore_hidden_elements` config](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara/Node/Finders#all-instance_method), which is [`true` by default](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara#configure-class_method) (and we haven't overridden it).

_TLDR: the `visible` option is `true`/`:visible` by default_